src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -164,6 +183,21 @@ private enum Flags : byte
             }
         }
 
+        internal void CreateBackingField(bool isCreatedForFieldKeyword)
+        {
+            lock (_backingFieldLock)
+            {
+                Debug.Assert(BackingField is null && !IsIndexer);
+                _lazyBackingFieldSymbol = new SynthesizedBackingFieldSymbol(this,
```

Comment:
>_lazyBackingFieldSymbol [](http://example.com/codeflow?start=16&length=23)

We are still unconditionally overwriting possibly non-null value.

Reply:
Should be fixed now.